### ISSUE TITLE
Non Ascii characters wrong in Windows installer

### ISSUE THREAD

**Issue by Rumbah2** [UltraStar-Deluxe/USDX#878]
### Actual behaviour

At least with German language, the windows installer of USDX does not display Umlauts correctly. They are just black boxes with a question mark inside.

![Unbenannt](https://github.com/user-attachments/assets/4dd4fb25-36e9-4080-8db9-cd9b7e70f08c)

### Expected behaviour

The Umlauts are displayed correctly, like äöüßÄÖÜ

### Steps to reproduce

1. Start the installer of USDX 2024.5.1 for Windows on a German Windows 11
2. You can see the wrong Umlauts

### Details

Provide some additional information:

- USDX version: 2024.5.1 Windows installer
- Operating System + version: Windows 11

**Comment by barbeque-squared** [comment]
These texts are coming from https://github.com/UltraStar-Deluxe/USDX/blob/master/installer/languages/German.nsh#L78 where they _do_ have umlauts.

Iirc I cannot debug/run the installer at all on my Windows PC.

(if it was only one or two I'd probably just 'fix' it by changing the words to not have special characters, but they're all throughout the installer and I have a suspicion this might also affect other languages then)

**Comment by DeinAlptraum** [comment]
It seems our installer is missing the `Unicode true` flag, but adding it produces a broken installer (`Unable to elevate`) ...

**Comment by dgruss** [comment]
<img width="499" height="388" alt="Image" src="https://github.com/user-attachments/assets/6ac8b20d-4041-4b59-9657-cc273bcad8cb" />

seems to work on the artifact from #1170 - haven't tried on the current master

**Comment by DeinAlptraum** [comment]
The issue still persists. This screenshot is from the last release artifact:

<img width="502" height="399" alt="Image" src="https://github.com/user-attachments/assets/3cb41fb8-b73b-43e8-a464-64c8c06b6e4c" />

(for reference, I have the Windows display language set to English (UK) and the "Language for non-Unicode programs" to Japanese. The German language pack is installed)

**Comment by dgruss** [comment]
hmm but it works on my Windows (German default installation, nothing special). Are we sure this is not some issue where Windows gets confused by unusual language settings?

**Comment by DeinAlptraum** [comment]
Yes I'm pretty sure this working or not is dependant on locale settings.
Probably only works with German locale atm, though I haven't tested that

**Comment by dgruss** [comment]
but that might be an issue outside of the game then, if the german installer on a non-german locale windows installation does not find the right characters, that seems outside of the scope of what we should handle.
since it works totally fine on german locale installation, i would not consider this a bug that we need to take care of.
